mltrace/cli/cli.py

```diff
@@ -307,3 +338,67 @@ def review(limit: int = 5, address: str = ""):
     # Print component runs
     for component, count in component_counts[:limit]:
         show_info_card(component.id, count, len(outputs))
+
+
+@mltrace.command("components")
+@click.option("--owner", help="Owner of components")
+@click.option("--tag", help="Tag of components")
+@click.option("--address", help="Database server address")
+def components(owner: str = "", tag: str = "", address: str = ""):
+    """
+    Command to list the components with options to filter by tag or owner.
+    """
+    if address and len(address) > 0:
+        set_address(address)
+
+    # Make return result
+    components = []
+    if owner:
+        components = get_components_with_owner(owner)
+    elif tag:
+        components = get_components_with_tag(tag)
+    else:
+        run_ids = get_recent_run_ids()
+        for run_id in run_ids:
+            cr_info = get_component_run_information(run_id)
+            c_info = get_component_information(cr_info.component_name)
+            components += [
+                {
+                    "name": c_info.name,
+                    "description": c_info.description,
+                    "owner": c_info.owner,
+                    "tags": c_info.tags,
+                }
+            ]
+
+    # Display components, one per line
+    for comp in components:
+        click.echo(f"Name: {comp.name}")
+        click.echo(f"└─Description: {comp.description}")
+        click.echo(f"└─Owner: {comp.owner}")
+        click.echo(f"└─Tags: {comp.tags}")
+        click.echo()
+
+
+@mltrace.command("tags")
+@click.option("--address", help="Database server address")
+def tags(address: str = ""):
+    """
+    Command to set the flag property of an output_id to true.
+    """
+    # Set address
+    if address and len(address) > 0:
+        set_address(address)
+
+    # Get all ids
+    component_run_ids = get_all_run_ids()
```

Comment:
Not an efficient way to do this. Querying the db directly for tags will be faster. To do this:

1) Make a method in `store.py` to query the tags

```
def get_all_tags(self) -> typing.List[Tag]:
  return self.session.query(Tag).all()
```

2. Create wrapper in `client.py` and whitelist the wrapper in `mltrace/__init__.py`

```
def get_all_tags()  -> typing.List[Tag]:
  store = Store(_db_uri)
  res = store.get_all_tags()
  # Convert res to client-side Tag objects & return
```

3. Use `client.py` function in `cli.py`

Reply:
done